tests/foreman/longrun/test_inc_updates.py

```diff
@@ -212,3 +214,110 @@ def test_positive_noapply_api(
         outval['action']
         == 'Incremental Update of 1 Content View Version(s) with 1 Package(s), and 1 Errata'
     )
+
+
+@pytest.mark.tier3
+def test_positive_incremental_update_time(module_target_sat, module_sca_manifest_org):
+    """Incremental update should not take a long time.
+
+    :id: a9cdcc58-2d10-42cf-8e24-f7bec3b79d6b
+
+    :steps:
+        1. Setup larger rh repositories; rhel8 baseOS, rhst8, rhsc8.
+        2. Create content view and add repos, sync and wait.
+        3. Publish a content view version with all content.
+        4. Using hammer, perform incremental update with errata, on that new version.
+            - Log the duration of the incremental update
+        5. Publish the full content-view, with added incremental version.
+            - Log the duration of the content-view publish
+
+    :expectedresults:
+        1. Incremental update takes a short amount of time.
+        2. Incremental update takes less time than full content-view publish,
```

Comment:
How do we gauge a `less/short time here`? And shouldn't this test be the part of performance team ?

Reply:
@jyejare great question! in line with the BZs, I don't think is as much of a performance/benchmark check, just to check that the update did not take significantly longer than publish, or take significantly long in general, (unexpected per local and prt testing).

in PRT, for these repos and CV, consistently seeing about 10-11 seconds to incr. update, and then 11-13 seconds to publish. Rather than benchmarking performance, we just see that the duration was qual to or shorter than publish, or was only a bit longer (since often they can be close, if both only took a few seconds).

At the bottom of the case, the arbitrary timeout assertions could help discover a performance issue- but it is pretty shallow in this regard, not meant to test different areas of performance in depth with currently implementation

Reply:
Sure. I just wanted to make sure that we arent spending time onto something that's not our area of verification.